flink-runtime/src/test/java/org/apache/flink/runtime/testutils/InMemoryReporter.java

```diff
@@ -240,6 +252,15 @@ public void addToConfiguration(Configuration configuration) {
         configuration.setString(
                 ConfigConstants.METRICS_REPORTER_PREFIX + "mini_cluster_resource_reporter." + ID,
                 id.toString());
+        configuration.set(SCOPE_NAMING_JM_JOB, "<job_id>.<host>.jobmanager.<job_name>");
```

Comment:
if you rely on the variables map (see above) then this change is unnecessary

Reply:
Great! That's a much better approach!